src/app/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,45 @@
+import cn from "classnames";
+import React from "react";
+
+import { Toggle } from "../Toggle";
+import { SwitchText } from "./SwitchText";
+
+interface SwitchOption<TValue> {
+	label: string;
+	value: TValue;
+}
+
+export type SwitchOptions<TValue = string> = [SwitchOption<TValue>, SwitchOption<TValue>];
```

Comment:
here is good to use object with left/right option key
it's easier when use it and for maintenance in future like here:
https://github.com/ArkEcosystem/desktop-wallet/pull/3843/files#diff-8b63a76540644aa1d1b5d4de1bf27ff0c62d66c96c97d231f9a1113f1b042ef9R128-R137
```suggestion
export type SwitchOptions<TValue = string> = {leftOption: SwitchOption<TValue>, rightOption: SwitchOption<TValue>};
```

Reply:
@w3ea thanks, that is good point. I ended up adding `leftOption` and `rightOption` as props, also to reduce nesting. https://github.com/ArkEcosystem/desktop-wallet/pull/3843/commits/d1b89491d21e41e37475d5da15ecc8ba384c3111